master/pkg/searcher/random.go

```diff
@@ -8,10 +8,12 @@ import (
 )
 
 type (
-	// randomSearchState stores the state for grid. Though random is stateless, it is useful
+	// randomSearchState stores the state for random. Though random is stateless, it is useful
 	// on restart to know the type of searcher during restore, in the event it must be shimmed.
 	// This gives us the ability to differentiate single and random in a shim.
```

Comment:
blocking: same thing as for grid - does this change make it stateful/modify comment?

Reply:
thanks for catching this